packages/react-ui/components/Token/TokenView.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { ReactNode, useContext } from 'react';
+
+import { CommonWrapper } from '../../internal/CommonWrapper';
+import { ThemeContext } from '../../lib/theming/ThemeContext';
+import { cx } from '../../lib/theming/Emotion';
+
+import { TokenSize } from './Token';
+import { globalClasses, styles } from './Token.styles';
+
+export interface TokenViewProps extends React.HTMLAttributes<HTMLDivElement> {
+  size?: TokenSize;
+  children: ReactNode;
+  closeButton?: ReactNode;
+  className?: string;
+  onRemove?: React.MouseEventHandler<HTMLElement>;
+}
+
+export function TokenView(props: TokenViewProps) {
+  const { size = 'small', children, closeButton, className, ...rest } = props;
+  const theme = useContext(ThemeContext);
+  const getSizeClassName = (size: TokenSize) => {
+    switch (size) {
+      case 'large':
+        return styles.tokenLarge(theme);
+      case 'medium':
+        return styles.tokenMedium(theme);
+      case 'small':
+      default:
+        return styles.tokenSmall(theme);
+    }
+  };
+
+  return (
+    <CommonWrapper {...props}>
+      <div
+        className={cx(getSizeClassName(size), className, {
+          [styles.token(theme)]: true,
+        })}
+        {...rest}
+      >
```

Comment:
`className` из props фактически попадает на div два раза, из-за того, что CommonWrapper внутри его тоже извлекает и прокидывает на первый вложенный в него элемент. Лучше оставить эту работу CommonWrapper, и не добавлять className из пропов на div самим. А если еще перенести `{...rest}` повыше, то и извлекать className из пропов в самом начале не придется.
```suggestion
      <div
        {...rest}
        className={cx(getSizeClassName(size), styles.token(theme))}
      >
```